<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="A571LBLR" name="com.oops.concepts.LoginTest" tests="2" failures="0" timestamp="21 Mar 2021 17:08:38 GMT" time="39.693" errors="1">
  <testcase name="verifyLoginPageTitle" time="23.983" classname="com.oops.concepts.LoginTest"/>
  <testcase name="doLoginTest" time="15.710" classname="com.oops.concepts.LoginTest">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.oops.concepts.BasePage.getPageHeader(BasePage.java:23)
at com.oops.concepts.HomePage.getHomePageHeader(HomePage.java:21)
at com.oops.concepts.LoginTest.doLoginTest(LoginTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- doLoginTest -->
</testsuite> <!-- com.oops.concepts.LoginTest -->
